ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -343,3 +382,267 @@ Set RHODS Users Group To Inexistent Group
+    END
+    # By default are enabled
+    IF  ${enabled} != True
+        Click Element    ${ACCELERATOR_ENABLED_SWITCH}
+    END
+    # Tolerations
+    IF    "${tolerations}" != "${EMPTY}"
+        Click Button    ${ACCELERATOR_ADD_TOLERATION_BTN}
+        Wait Until Element Is Visible    ${ACCELERATOR_ADD_TOLERATION_MODAL_FORM}
+        # Select Operator
+        Click Button    ${ACCELERATOR_TOLERATION_OPERATOR_DROPDOWN}
+        Click Element    //div[@data-testid="toleration-operator-select"]//div[text()="${tol_operator}"]
+        # Select Effect
+        Click Button    ${ACCELERATOR_TOLERATION_EFFECT_DROPDOWN}
+        Click Element    //label[@for="effect-select"]/parent::*/following::div//*[text()="${tol_effect}"]
+        # Input Key and value
+        Input Text    ${ACCELERATOR_TOLERATION_KEY}    ${tol_key}
+        Input Text    ${ACCELERATOR_TOLERATION_VALUE}    ${tol_value}
+        # Toleration Seconds
+        IF    "${tol_seconds}" != "Forever"
+            Click Element    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_RADIO}
+            Clear element and input text    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_VALUE}   ${tol_seconds}
+        ELSE
+            Click Element    ${ACCELERATOR_TOLERATION_SECONDS_FOREVER_RADIO}
+        END
+        Click Button    ${ACCELERATOR_TOLERATION_ADD_OR_UPDATE_BUTTON}
+    END
+    Click Button    ${CREATE_OR_UPDATE_ACCELERATOR_PROFILE_DETAIL_BTN}
+
+Modify The Accelerator Profile
+    [Documentation]    Modify Accelerator Profile values with the ones provided in the arguments and click on
+    ...                "Update acccelerator profile" button
+    ...                view: Edit Accelerator profile
+    [Arguments]   ${original_display_name}  ${display_name}=${EMPTY}  ${identifier}=${EMPTY}  ${description}=${EMPTY}
+    ...           ${enabled}=${EMPTY}  ${tolerations}=${EMPTY}  ${tol_operator}=${EMPTY}  ${tol_effect}=${EMPTY}
+    ...           ${tol_key}=${EMPTY}  ${tol_value}=${EMPTY}  ${tol_seconds}=${EMPTY}
+    Wait Until Element Is Visible   //h1[text()="Edit ${original_display_name}"]  timeout=10
+    IF    "${display_name}" != "${EMPTY}"
+        Clear element and input text    ${ACCELERATOR_NAME}    ${display_name}
+    END
+    IF    "${identifier}" != "${EMPTY}"
+        Clear element and input text    ${ACCELERATOR_IDENTIFIER}    ${identifier}
+    END
+    IF    "${description}" != "${EMPTY}"
+        Clear element and input text    ${ACCELERATOR_DESCRIPTION}    ${description}
+    END
+    # Click always, as there is no way from the UI to obtain the status of the switch,
+    IF    "${enabled}" != "${EMPTY}"
+        Click Element    ${ACCELERATOR_ENABLED_SWITCH}
+    END
+    IF    "${tolerations}" != "${EMPTY}"
+        ${first_kebab}=    Catenate    SEPARATOR=    ${ACCELERATOR_EDIT_TOLERATION_KEBAB_BTN}    [1]
+        Click Button    ${first_kebab}
+        Wait Until Element Is Visible    ${first_kebab}/following::div//span[text()="Edit"]    timeout=10
```

Comment:
this arbitrarly edits the first toleration, is that correct? Isn't there a way to change a specific toleration from the list?